src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/BaseOptions.java

```diff
@@ -165,6 +165,12 @@
     // A list of pairs containing urls and package list
     private final List<Utils.Pair<String, String>> linkOfflineList = new ArrayList<>();
 
+    /**
+     * Argument for command-line option {@code --link-modularity-mismatch}.
+     * True if warnings on external documentation with non-matching modularity should be omitted.
+     */
+    private boolean linkModularityNoWarning = false;
```

Comment:
Would it be better to use a local enum that matches the option values?

Reply:
I introduced an enum, I agree it's an improvement. Hope you are ok with the naming.